benchmark/buffers/buffer-tojson.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const common = require('../common.js');
+
+const bench = common.createBenchmark(main, {
+  n: [1e4],
+  len: [0, 10, 256, 4 * 1024]
+});
+
+function main(conf) {
+  var n = +conf.n;
```

Comment:
In addition to what @mscdex said, the current position on `const`/`let` over `var` is "only in code where performance doesn't matter". I'd say it makes more sense for `benchmarks/` to be consistent with `lib/` than `test/`.